src/content/blog/ga_qas.md

```diff
@@ -0,0 +1,224 @@
+---
+title: An Introduction to GA-QAS
+author: Vu Tuan Hai
+day: 21
+month: 5
+year: 2024
+tags: 
+  - quantum architecture search
+  - python
+  - qiskit
+  - genetic-algorithm
+---
+
+In various quantum optimization problems, choosing the right ansatz is a critical point that will affect the result. Many template ansatzes have been proposed, such as Graph Ansatz and EffecientSU2, … but they are limited to the applications that you can apply. Then, we propose a search engine called **GA-QAS** (Genetic algorithm for quantum architecture search). This search engine can take your problem as input and return a good ansatz for you. In this post, we will guide you on how to use it efficiently.
+
+## Now, let’s start!
+
+# Step 1. Define problem (W state preparation)
+
+<img alt ='' src="/public/images/ga-qas/ga-qas1.png">
```

Comment:
1. I think this should fix the broken image:
```suggestion
<img alt ='' src="/images/ga-qas/ga-qas1.png">
```

2. Please add some alternative text for the image in case someone is using a screen reader. A guide can be found [here](https://web.dev/articles/text-alternatives-for-images).